client/blocks/login/docs/example.jsx

```diff
@@ -11,7 +11,12 @@ import React from 'react';
  */
 import LoginBlock from 'blocks/login';
 
-const LoginExample = () => <LoginBlock title={ 'Sign in to connect to WordPress.com' } />;
+const LoginExample = () => (
+	<React.Fragment>
+		<LoginBlock />
+		<LoginBlock jetpack />
```

Comment:
The logo from the second `LoginBlock` bangs into the bottom of the first `LoginBlock` right now. Can we do anything to prevent that?

Reply:
I stuck a `<p />` in there which adds some margin 👍 
0338d18aeb6ff9d85bfc451708bc86b207e1a5a2